service/core.go

```diff
@@ -126,25 +126,25 @@ func (s *serviceCore) AuthClient() auth.Client {
 }
 
 func (s *serviceCore) Name() string {
```

Comment:
There are a lot of repos using `service_name` and new services are actually generated with `serviceName` instead of `applicationID`.